src/consts.ts

```diff
@@ -65,6 +65,7 @@ export const FallbackSessionsPerEra = 6;
 export const FallbackNominatorRewardedPerValidator = 256;
 export const FallbackMaxElectingVoters = 22500;
 export const FallbackExpectedBlockTime = 1000;
+export const FallbackExpectedEraTime = 24 * 60 * 60 * 1000;
```

Comment:
Please double check if this value is correct.

Reply:
I had that in mind as well, but wanted to stick to the `FallbackExpectedBlockTime` convention which does not do that. However, since you noticed it as well, it's a good sing that it's gotta be done ✔️